sascha_willems/README.md

```diff
@@ -0,0 +1,16 @@
+# Vulkan Samples Assets
+
+Licenses and attributions for the various scenes and textures used in this sub-folder.
+
+## Scenes
+
+- Buggy.gltf taken from the [Khronos gltf Sample Models repository](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Buggy).
+- All other gltf scenes have been created for this Repository by Sascha Willems and are licensed under [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+## Textures
+
+- Uffizi hdr environment map copyright by [USC Institute for Creative Technologies](http://gl.ict.usc.edu/Data/HighResProbes/)
```

Comment:
Also the binary files won't let me comment on them so here it goes.

1) What did we decide on the structure of this commit? should we rename `sascha_willems/scenes` into `scenes`
2) Are the textures compressed? should they be?
3) Are there any textures used by multiple of your glTF assets?

Reply:
> What did we decide on the structure of this commit? should we rename sascha_willems/scenes into scenes

Yes, this makes sense now. Will move my scenes to the scenes folder and adjust code. Will also move textures into textures folder once you greenlit my texture naming scheme.

> Are the textures compressed? should they be?

Right now all textures are non-compressed RGBAs in order to avoid having to multiply them for different compression formats (BC, ASTC, ETC). 

>Are there any textures used by multiple of your glTF assets?

Not yet.

Reply:
Thanks,
I am happy with the texture naming.